src/common/cssVariables.js

```diff
@@ -0,0 +1,34 @@
+export const blue_light = 'rgba(173, 216, 230, 1)';
+export const grey_dark = 'rgba(128, 128, 128, 1)';
+export const blue_medium = 'rgba(51, 102, 255, 0.5)';
+export const red_dark = 'rgba(255, 0, 0, 1)';
+export const blue_dark = 'rgba(0, 0, 255, 1)';
+export const black_light = 'rgba(51, 51, 51, 1)';
+export const grey_light = 'rgba(107, 128, 150, 1)';
+export const white = 'rgba(	250, 250, 250, 1)';
+export const temperatureTextColor = red_dark;
+export const windTextColor = blue_dark;
+export const buttonBackgroundColorLight = blue_medium;
+export const buttonBackgroundColorDark = grey_light;
+export const footerBackgroundColorLight = blue_light;
+export const footerBackgroundColorDark = grey_dark;
+export const headerBackgroundColorLight = blue_light;
+export const headerBackgroundColorDark = grey_dark;
+export const filterableListBackgroundColorLight = blue_light;
+export const filterableListBackgroundColorDark = grey_dark;
+export const aTextColorLightTheme = black_light;
+export const aTextColorDarkTheme = white;
+export const textColorDarkTheme = white;
+export const textColorLightTheme = black_light;
+export const toggleBorderLight = white;
```

Comment:
could you explain why is this variable named somethingLight and the previous is somethingLightTheme ? What is the conceptual difference between them so that you use Theme word for one and don't use it for another ?

Reply:
Anton, again... 

[Rebels Alliance Son](https://www.youtube.com/watch?v=NoXZZhgUo_0)